e2e/build_uki_test.go

```diff
@@ -52,6 +52,26 @@ var _ = Describe("build-uki", func() {
 		})
 	})
 
+	Describe("single-efi-cmdline", func() {
+		BeforeEach(func() {
+			By("building the iso with single-efi-cmdline flags set")
+			buildISO(enki, image, keysDir, resultDir, resultFile,
+				"--single-efi-cmdline", "My Entry: someoption=somevalue",
+				"--single-efi-cmdline", "My Other Entry: someoption2=somevalue2")
+		})
+
+		It("creates additional .efi and .conf files", func() {
+			By("checking if the default value for secure-boot-enroll is set")
+			content := listEfiFiles(enki, resultFile)
+			Expect(string(content)).To(MatchRegexp("My_Entry.efi"))
```

Comment:
wondering if when handling the efi files later on the agent, e.g.. on upgrade, it would be easier/simpler to always use the same name as Itxaka was pointing out cusom1.efi custom2.efi

Reply:
Currently, there is not change needed at the agent. It just works. So there is no benefit in naming them "custom1.efi`, `custom2.efi` etc. Also the way they are named now, it's easier to tell which one is which.

Reply:
yup good point